bundled/setup_bundled.cmake

```diff
@@ -78,6 +78,9 @@ set(KOKKOS_FOLDER "${CMAKE_SOURCE_DIR}/bundled/kokkos-3.7.00")
 macro(feature_kokkos_configure_bundled)
   set(Kokkos_VERSION "3.7.0")
   set(KOKKOS_VERSION "3.7.0")
+  set(KOKKOS_VERSION_MAJOR "3")
```

Comment:
Why do you need this considering you are parsing the version string?

Reply:
You can end up in the `configure_bundled` macro without ever calling the `FindDEAL_II_KOKKOS` module.

Reply:
Yes, indeed. That's how I found this place: When we fall back to the bundled boost, we don't call the other place that parses the version number.